doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -219,6 +219,17 @@ Asset Tracker v2
   * The :kconfig:option:`NRF_CLOUD_SEND_SERVICE_INFO_FOTA` and :kconfig:option:`NRF_CLOUD_SEND_SERVICE_INFO_UI` Kconfig options.
     The application no longer sends a device shadow update; this is now handled by the :ref:`lib_nrf_cloud` library.
 
+* Updated:
+
+  * Few power optimizations to LwM2M configuration overlay:
+
+    * Enable DTLS Connection Identifier.
+    * Do LwM2M update once a hour and try to request similar periodic tracking area update intervall from LTE network.
```

Comment:
```suggestion
    * Perform LwM2M update once an hour and request for similar update interval of periodic tracking area from the LTE network.
```
I may have messed up this big time. However this needs some attention. The long list of substantives "tracking area update interval" bothered me a bit.